modules/db/src/test/resources/integration/select/select-unresolved-named-param-config.xml

```diff
@@ -8,6 +8,7 @@
     <flow name="unresolvedNamedParam">
         <db:select>
             <db:parameterized-query>select * from Planet where name = :name</db:parameterized-query>
+            <db:in-param name="name" value="#[payload]"/>
```

Comment:
This case is testing the definition of a db:in-param element with a name that isn't referenced by the query text.
Due to the changes, As text query parameter references an inexistent db-in-param element, this exception is triggered first.
So I added a db:in-param with name "name" to can test this feature.